### ISSUE TITLE
Possibility to attach pictures to notes

### ISSUE THREAD

**Issue by thigg** [owncloud/notes#224]
It would be nice to be able to attach pictures to notes.

**Comment by schannall** [comment]
Adding binary Data as Attachments would be nice (e.g. attach:img02.jog or attach:recording1.mp3) and storing this beside the note would be great!

sC

**Comment by cengbrecht** [comment]
Enhancement request! Please! :-)

**Comment by djm193** [comment]
yeah, adding photos to Notes would be really nice! With this feature you could skip iOS notes completly

**Comment by HeaDBanGer84** [comment]
Markdown already supports inserting images. You just need a concept for storage and syncing/API

**Comment by Whisprin** [comment]
To get started the webinterface can just support the markdown image syntax. We would still have to host the image ourself. 

Later the image can get uploaded to a special subdirectory within owncloud and shared using the already existing tools. A concept for storage and sync is precisely what owncloud already offers. Would be nice to re-use that.

**Comment by eemantsal** [comment]
Any news from april 2016? Nowadays we take notes with much more content than just text, especially since we all have mobile devices with cameras. I think is a rather basic feature.
There's a desktop application for creating MD notes, [Qownnotes](https://github.com/pbek/QOwnNotes), that creates a file named "Media" inside the folder of the note and saves there all the pictures we embed; then, in the note, there's just a link to the corresponding file in that "Media" folder. Seems rather uncomplicated (from a non programmer point of view, heheh).
